src/main/scala/li/cil/oc/server/component/WirelessNetworkCard.scala

```diff
@@ -58,36 +52,49 @@ class WirelessNetworkCard(host: EnvironmentHost) extends NetworkCard(host) with
   }
 
   // ----------------------------------------------------------------------- //
-
+  
   @Callback(direct = true, doc = """function():number -- Get the signal strength (range) used when sending messages.""")
   def getStrength(context: Context, args: Arguments): Array[AnyRef] = result(strength)
 
   @Callback(doc = """function(strength:number):number -- Set the signal strength (range) used when sending messages.""")
   def setStrength(context: Context, args: Arguments): Array[AnyRef] = {
-    strength = math.max(0, math.min(args.checkDouble(0), Settings.get.maxWirelessRange))
+    strength = math.max(0, math.min(args.checkDouble(0), maxWirelessRange))
     result(strength)
   }
 
   override def isWireless(context: Context, args: Arguments): Array[AnyRef] = result(true)
-
+  
+  override def isWired(context: Context, args: Arguments): Array[AnyRef] = result(shouldSendWiredTraffic)
+
+  override def open(context: Context, args: Arguments): Array[AnyRef] = {
```

Comment:
could we instead share the same open check (i.e. NOT duplicate this method, and instead override the max ports in this wireless class?